test/unit/object/test_task_assignment.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import unicode_literals
+
+import json
+import pytest
+
+from mock import Mock
+from boxsdk.config import API
+from boxsdk.object.task_assignment import TaskAssignment
+from boxsdk.network.default_network import DefaultNetworkResponse
+
+
+@pytest.fixture(scope='module')
+def delete_task_assignment_response():
+    # pylint:disable=redefined-outer-name
+    mock_network_response = Mock(DefaultNetworkResponse)
+    mock_network_response.ok = True
+    return mock_network_response
+
+
+def test_get_assignment(test_task_assignment, mock_box_session):
+    expected_url = '{0}/task_assignments/{1}'.format(API.BASE_API_URL, test_task_assignment.object_id)
+    mock_box_session.get.return_value.json.return_value = {
+        'type': test_task_assignment.object_type,
+        'id': test_task_assignment.object_id,
+        'assigned_to': {
+            'type': 'user',
+            'id': '11111',
+        },
+    }
+    retrieved_task = test_task_assignment.get()
+    mock_box_session.get.assert_called_once_with(expected_url, headers=None, params=None)
+    assert isinstance(retrieved_task, TaskAssignment)
+    assert retrieved_task.object_type == test_task_assignment.object_type
+    assert retrieved_task.object_id == test_task_assignment.object_id
+    assert retrieved_task.assigned_to['type'] == 'user'
+    assert retrieved_task.assigned_to['id'] == '11111'
+
+
+def test_delete_policy_return_the_correct_response(
+        test_task_assignment,
+        mock_box_session,
+        delete_task_assignment_response,
+):
+    # pylint:disable=redefined-outer-name
+    mock_box_session.delete.return_value = delete_task_assignment_response
+    response = test_task_assignment.delete()
+    # pylint:disable=protected-access
+    expected_url = test_task_assignment.get_url()
```

Comment:
This URL should be hardcoded to ensure the test actually checks that the implementation is correct

Reply:
This URL should also be hardcoded